src/helpers/Api/APIResourceWithAuth.js

```diff
@@ -21,7 +21,7 @@ export const getAPIResourceWithAuth = (req) =>
     let apiPath = '';
     if (settings.internalApiPath && __SERVER__) {
       apiPath = settings.internalApiPath;
-    } else if (__DEVELOPMENT__ && settings.devProxyToApiPath) {
```

Comment:
Looks like this one may have broken a test in seamless mode; I need to take a closer look.

Reply:
@sneridagh Without the check for `__DEVELOPMENT__`, this ends up trying to use the default devProxyToApiPath (http://localhost:8080/Plone) in seamless mode in production, instead of leaving the apiPath blank.

I think there is probably a way to make this less confusing by using a default for internalApiPath instead of devProxyToApiPath, but I'm going to take this change out of this PR for now.